app/src/main/java/com/puzzlebench/clean_marvel_kotlin/data/local/SaveCharactersLocalImpl.kt

```diff
@@ -11,7 +11,5 @@ class SaveCharactersLocalImpl(private val mapper: CharacterMapperLocal = Charact
         realm.beginTransaction()
```

Comment:
Try to use executeTransaction(), as it is less error prone. 
More information here: https://github.com/realm/realm-java/issues/4217

Reply:
done